frontend/src/app/utils/resource-utils.js

```diff
@@ -19,6 +19,11 @@ const hostsPoolModeToStateIcon = deepFreeze({
         css: 'warning',
         name: 'working'
     },
+    INITIALIZING_FAILED: {
+        tooltip: 'Initialization Failed',
```

Comment:
I am not sure this string can occupy the given space we have for the text. Please test with minimal UI width

Reply:
there are many longer tooltips in the same table